File: Programming in C/numberPalindrome.c
#include <stdio.h>
#include <stdbool.h>
int reverseNumber(int n)
{
    int temp = n;
    int reverseNumber = 0;
    while (temp)
    {
        reverseNumber = (reverseNumber * 10 + temp % 10);
        temp = temp / 10;
    }
    return reverseNumber;
}
bool checkPlaindrome(int n)
{
    if (reverseNumber(n) == n)
        return true;
    else
        return false;
}
int main()
{
    int n = 0;
    printf("Enter the number : ");
    scanf("%d", &n);
    if (checkPlaindrome(n))
        printf("%d is a palindrome\n", n);
    else
        printf("%d is not a plaindrome\n", n);
    return 0;
}

File: WebApplicationDevelopment/Cycle4/numberPalindrome.js
function reverseNumber(number)
{
    let orginalNumberArray = number.split("");
    let reversedNumberArray = orginalNumberArray.reverse();
    let finalNumber = reversedNumberArray.join("");
    if (finalNumber == number)
        return true;
    else 
        return false;
}


let number = prompt("Enter a five digit number : ");
let isPalindrome = reverseNumber(number);
if(isPalindrome == true)
    console.log("The number is a plaindrome");
else
    {
        number = prompt("The number was not a plaindrome. Enter a five digit number : ");
        isPalindrome = reverseNumber(number);
        if(isPalindrome == true)
            console.log("The number is a plaindrome");
        else
            console.log("The number is not a plaindrome.")
    }